content/en/docs/tasks/manage-gpus/scheduling-gpus.md

```diff
@@ -156,6 +156,39 @@ kubectl label nodes <node-with-k80> accelerator=nvidia-tesla-k80
 kubectl label nodes <node-with-p100> accelerator=nvidia-tesla-p100
 ```
 
+For AMD GPUs, you can deploy [Node Labeller](https://github.com/RadeonOpenCompute/k8s-device-plugin/tree/master/cmd/k8s-node-labeller), which automatically label your nodes with GPU properties.  Currently supported properties are:
+
+* Device ID (-device-id)
+* VRAM Size (-vram)
+* Number of SIMD (-simd-count)
+* Number of Compute Unit (-cu-count)
+* Firmware and Feature Verions
+* GPU Family, in two letters acronym (-family)
```

Comment:
nit: Is it Firmware and Feature Versions?

Reply:
Yes.  It is both.  But I noticed I missed a flag there.

Reply:
spelling: Verions?

Reply:
Oh right... sorry I missed that.